packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.component.js

```diff
@@ -4,6 +4,8 @@ import template from './traffic.html';
 export default {
   bindings: {
     getSubnet: '<',
+    ip: '<',
+    dateTime: '<',
```

Comment:
```suggestion
```
No need because you call directly $stateParams in your controller also ip and dataTime doesn't injected in the routing.